.github/tl_packages

```diff
@@ -12,7 +12,11 @@ tex
 xetex
 # Requirements for the tests
 amsfonts
+dvips
 etex-pkg
+l3experimental
+latex-lab
+pdfmanagement-testphase
```

Comment:
Will these three packages too experimental to cause less stable test output files?